toolkit/components/uniffi-bindgen-gecko-js/src/render/cpp.rs

```diff
@@ -58,7 +58,7 @@ pub impl FFIFunction {
     }
 
     fn rust_name(&self) -> String {
-        self.name().to_snake_case()
+        self.name().to_string()
```

Comment:
this shouldn't be snake_case since methods on interface usually have their interface name capitalized... we should change https://github.com/mozilla/uniffi-rs/blob/main/uniffi_bindgen/src/scaffolding/templates/ObjectTemplate.rs#L64 first if we want it to be proper snake_case